staging/src/k8s.io/apiserver/pkg/util/flowcontrol/exempt_borrowing_test.go

```diff
@@ -104,9 +108,9 @@ func TestUpdateBorrowing(t *testing.T) {
 	expectedExempt := serverCL - (expectedHigh + expectedMid + expectedLow)
 	stateExempt.seatDemandIntegrator.Set(float64(expectedExempt))
 	clk.SetTime(startTime.Add(2 * borrowingAdjustmentPeriod))
-	ctlr.updateBorrowing()
+	ctlr.updateBorrowing(ctx)
 	clk.SetTime(startTime.Add(3 * borrowingAdjustmentPeriod))
-	ctlr.updateBorrowing()
+	ctlr.updateBorrowing(context.Background())
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
done